snap/container.go

```diff
@@ -37,6 +37,7 @@ type Container interface {
 	RandomAccessFile(relative string) (interface {
 		io.ReaderAt
 		io.Closer
+		Stat() (os.FileInfo, error)
```

Comment:
we really want just Size here I think (too many details of a full stat are not well defined because they relate to a temp file in some implementations)

Reply:
Changed implementation to provide only a Size() call.